prijateli_tree/app/config.py

```diff
@@ -14,7 +14,7 @@
 class BaseConfig:
     def __init__(self):
         # App Settings
-        self.DEBUG = False
+        self.DEBUG = True
```

Comment:
We want to keep this set to False, because it is inherited by all other configurations and DEBUG is only used for development.